Fix: TrueNAS Core support for pool stats (#3206)
gethomepage/homepage: docs/widgets/services/truenas.md

A detailed pool listing is disabled by default, but can be enabled with the `enablePools` option.

> TrueNAS Scale and Core APIs differ, the optional `nasType` parameter is needed if using the `enablePools` option.
> 

Reviewer: ```suggestion
To use the `enablePools` option with TrueNAS Core, the `nasType` parameter is required.

```

  password: pass # not required if using api key
  key: yourtruenasapikey # not required if using username / password
  enablePools: true # optional, defaults to false

>   nasType: scale # defaults to scale, values are core or scale

Reviewer: ```suggestion
  nasType: scale # defaults to scale, must be set to 'core' if using enablePools with TrueNAS Core
```
